### PR TITLE
Increase Idea Title Character Limit to 150

### PR DESCRIPTION
**Pull request by alanlgirard**
Increased the idea title character limit to 150 and fixed some over-flow/word wrapping issues

Closes issue  #254 

Simple review @YashdalfTheGray @Mctalian @monotkate @davethenipper

### CODE REVIEW

File: src/assets/app.css [hunk 1]
```diff
@@ -298,16 +298,15 @@ h2.no-bottom-margin {
...
 
 .word-wrap {
-    -ms-word-break: normal;
-    word-break: normal;
-    word-break: normal;
-    overflow-wrap: normal;
-    word-wrap: normal;
+    -ms-word-break: break-all;
+    word-break: break-all;
+    word-break: break-word;
```
Comment (Mctalian) [inline src/assets/app.css]: So this causes some weird behavior in the description. Words like crazy get broken on the end of the line. I was the one that changed it to normal, where were we seeing undesirable wrapping?

File: src/ideas/ideaInput/ideaInput.tpl.html [hunk 2]
```diff
@@ -2,12 +2,13 @@
     <div layout="column">
         <md-input-container>
             <label>Title</label>
-            <input ng-model="idea.title"
+            <textarea ng-model="idea.title"
```
Comment (Mctalian) [inline src/ideas/ideaInput/ideaInput.tpl.html]: It's big enough to need a text area now?! Haha, that might be too big. I just noticed 30 was always a lot shorter than I anticipated. I'm fine with whatever the other reviewers are fine with. I defer to better judgement.

Comment (alanlgirard) [inline src/ideas/ideaInput/ideaInput.tpl.html]: I switched it to a textarea for smaller resolutions where 150 characters will wrap.

Comment (Mctalian) [inline src/ideas/ideaInput/ideaInput.tpl.html]: :+1:

File: src/assets/app.css (hunk 1 above)
Comment (alanlgirard) [inline src/assets/app.css]: Didn't check IE.  The issue was if a title was 150 characters without a space it wasn't being broken up properly. That is why I put in the break-word. I will look into this tonight.  I hate having to do cross-browser support

**Comment by YashdalfTheGray** [comment]
This seems good, approved!

**Comment by Mctalian** [comment]
Approved. When I get into the office I'm going to pull all of this stuff down.

**Pull request merged**